Sources/TuistKit/GraphMappers/ProjectMapperProvider.swift

```diff
@@ -29,6 +29,13 @@ final class ProjectMapperProvider: ProjectMapperProviding {
             mappers.append(SynthesizedResourceInterfaceProjectMapper())
         }
 
+        // Logfile noise suppression
+        if config.generationOptions.contains(.disableShowEnvironmentVarsInScriptPhases) {
```

Comment:
I'd add a simple test case similar to `test_mappers_does_not_returns_resources_namespace_project_mapper_when_disabled_autogenerated_namespace` in `ProjectMapperProviderTests`

Reply:
Done